Add instructions for changing billing plans (#1600)
balena-io/docs: pages/learn/manage/billing.md

# Billing

Billing is managed through [organizations][organizations]. Navigate to the organization for which you'd like to manage billing, and select "billing" from the left hand side menu.

> <img width="189" alt="Screen Shot 2021-03-01 at 4 46 36 PM" src="https://user-images.githubusercontent.com/20308284/109579511-e54ea700-7aad-11eb-9cf7-5648f1fdb897.png">

Reviewer: I see now that this page inherently used the HTML tags way of adding images. Will find them all and replace. A possible reason to use HTML image tags is tinker with the width of a large image so that it fits right.

From this dashboard, you can change your billing plan, update your billing information, monitor your usage, and download invoices. If you have any questions about billing for your account, please contact [our customer success team][solutions].

## Changing your billing plan

Changing your billing plan is self-serve on balenaCloud. If you'd like to upgrade to a paid plan, select "change plan" from the billing page: 

<img width="1208" alt="Screen Shot 2021-03-01 at 4 59 54 PM" src="https://user-images.githubusercontent.com/20308284/109580811-17f99f00-7ab0-11eb-992b-f3993060a808.png">

You'll be shown the available plan options; select which plan you'd like to upgrade to: 

<img width="1434" alt="Screen Shot 2021-03-01 at 5 02 05 PM" src="https://user-images.githubusercontent.com/20308284/109580819-1c25bc80-7ab0-11eb-8c86-57e54547ddfe.png">

Enter your billing information and complete the signup process: 

<img width="1437" alt="Screen Shot 2021-03-01 at 5 01 48 PM" src="https://user-images.githubusercontent.com/20308284/109580829-2051da00-7ab0-11eb-924a-ca86d0e187a8.png">

> If you need to downgrade or cancel your plan, you can follow the same steps above. Just move to a lower tier or free plan instead. 

Reviewer: ```suggestion
If you need to downgrade or cancel your plan, you can follow the steps mentioned above by choosing a lower tier or free plan instead. 
```
I try to reduce the usage of `just` in documentation as it unconsciously adds a layer of ease to a step that might not be as easy for some of our users. This way our documentation stays inclusive.

## Dynamic billing

> For any usage that is outside the usage included in your billing plan, we will track your usage, and you will be billed dynamically in arrears. Dynamic billing for devices and users outside of your plan will occur every month, even for those on annual plans.

Reviewer: ```suggestion
Any usage that is outside the limits/offerings of the current billing plan, will be tracked and billed dynamically in arrears. Dynamic billing for devices and users outside of the organization's plan will occur every month, even for those with annual billing plans.
```

Author: I purposefully removed the word limits on this one. So I'll find a word that isn't limits but perhaps is clearer.
